scripts/run-integration-tests.sh

```diff
@@ -66,15 +68,22 @@ patch_excluded_tests() {
     fi
 }
 
-build_and_install_driver() {
-    log_info "Building exasol driver $driver_jar..."
-    local plugin_dir="$metabase_dir/plugins"
-    cd "$exasol_driver_dir"
-    clojure -X:build :project-dir "\"$(pwd)\""
-    ls -lah "$driver_jar"
-    log_info "Installing exasol driver $driver_jar to $plugin_dir"
-    mkdir -p "$plugin_dir"
-    cp -v "$driver_jar" "$plugin_dir"
+install_jdbc_driver() {
+    if [ ! -d "$metabase_plugin_dir" ]; then
+        log_info "Creating $metabase_plugin_dir"
+        mkdir -p "$metabase_plugin_dir"
+    fi
+
+    local exasol_driver="$metabase_plugin_dir/exasol-jdbc.jar"
```

Comment:
Extract driver file name to constant above.

Reply:
Done